terraform/modules/emr/local.tf

```diff
@@ -12,6 +12,29 @@ locals {
   autoscaling_max_capacity        = 5
   dks_port                        = 8443
   full_proxy                      = var.internet_proxy["http_address"]
+
+  configurations_mysql_json = templatefile(format("%s/templates/emr/configuration.mysql.json", path.module), {
+    logs_bucket_path             = format("s3://%s/logs", var.log_bucket)
+    data_bucket_path             = format("s3://%s/data", aws_s3_bucket.emr.id)
+    notebook_bucket_path         = format("%s/data", aws_s3_bucket.emr.id)
```

Comment:
I cant see where this is referenced in mysql.json file

Reply:
This is probably something we removed for the configuration.json a long time ago but forgot to remove the variable.